test/judges/codechef/getContests.test.js

```diff
@@ -0,0 +1,74 @@
+console.log = s => {
+  process.stdout.write(s + '\n');
+};
+
+import { getContests } from '../../../src/judges/codechef/getContests';
+import { JSDOM } from 'jsdom';
+import * as MockDate from 'mockdate';
+
+MockDate.set('5/8/2018');
+
+JSDOM.fromURL = jest.fn().mockReturnValue(JSDOM.fromFile('test/resources/codechef.html'));
```

Comment:
Should we open an issue to discuss the use/lack of flow on test files?

Reply:
I think not. Maybe I am horribly wrong but I don't see the point of type checking our tests.

Reply:
I don't think we should type them either.